From 59d2f8e59bd1612c13b093d51b500f90635a7f97 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Date: Fri, 15 May 2020 16:11:04 +0300
Subject: [PATCH 123/269] Revert "s32g: ssram_bl1: Re-enable DDR and jump to
 BL31"

This reverts commit ddc201cf149108015de70a5946972e21c0bb5575.

Issue: ALB-4907
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/s32g/platform.mk      |  5 +----
 plat/s32g/s32g_ssram_bl1.c | 16 ----------------
 2 files changed, 1 insertion(+), 20 deletions(-)

diff --git a/plat/s32g/platform.mk b/plat/s32g/platform.mk
index e96d699c2..ae86e30e3 100644
--- a/plat/s32g/platform.mk
+++ b/plat/s32g/platform.mk
@@ -36,10 +36,7 @@ PLAT_BL_COMMON_SOURCES	+= plat/s32g/s32g_lowlevel_common.S \
 			   drivers/delay_timer/generic_delay_timer.c \
 			   lib/cpus/aarch64/cortex_a53.S
 
-BL1_SOURCES		+= plat/s32g/s32g_ssram_bl1.c \
-			   plat/s32g/s32g_clocks.c \
-			   ${DDR_DRV}/ddrss.c \
-			   ${DDR_DRV}/ddrss_regconf.c
+BL1_SOURCES		+= plat/s32g/s32g_ssram_bl1.c
 
 BL2_SOURCES		+= plat/s32g/s32g_lowlevel_bl2.S \
 			   plat/s32g/s32g274a_bl2_el3.c \
diff --git a/plat/s32g/s32g_ssram_bl1.c b/plat/s32g/s32g_ssram_bl1.c
index 0a4cad4cb..bf04ff754 100644
--- a/plat/s32g/s32g_ssram_bl1.c
+++ b/plat/s32g/s32g_ssram_bl1.c
@@ -5,21 +5,6 @@
  */
 
 #include <plat/common/platform.h>
-#include <common/debug.h>
-#include <s32g_clocks.h>
-#include <drivers/nxp/s32g/ddr/ddrss.h>
-#include <platform_def.h>
-
-void  __attribute__ ((noreturn)) s32g_system_resume(void)
-{
-	extern struct ddrss_conf ddrss_conf;
-	void (*bl31_entrypoint)(void) = (void (*)(void))BL31_BASE;
-
-	s32g_plat_ddr_clock_init();
-	ddrss_to_normal_mode(&ddrss_conf);
-	bl31_entrypoint();
-	panic();
-}
 
 int plat_get_image_source(unsigned int image_id, uintptr_t *dev_handle,
 			  uintptr_t *image_spec)
@@ -63,7 +48,6 @@ void platform_mem_init(void)
 
 void bl1_early_platform_setup(void)
 {
-	s32g_system_resume();
 }
 
 void bl1_plat_arch_setup(void)
-- 
2.17.1

